test/unit/log.provider.test.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {sinon} from '@loopback/testlab';
+import {ParsedRequest} from '@loopback/core';
+import {LogProvider} from '../..';
+
+describe('LogProvider', () => {
```

Comment:
`describe('LogProvider (unit)')`

the test file should be `test/unit/providers/log-action.provider.test.ts` (or `...unit.ts`)

See also https://github.com/strongloop/loopback-next-extension-starter/pull/6#discussion_r142387171 and https://github.com/strongloop/loopback-next-extension-starter/pull/6#discussion_r142387402